### PR TITLE
Update Avalonia to 0.9.12 so it can work with ubuntu 20.04

### PR DESCRIPTION
**Pull request by mimran1980**
Getting seg faults in ubuntu 20.04, updated avalonia which seems to resolve this issue.

### CODE REVIEW

**Comment by olivierchoquet** [comment]
I confirm that the version of mimran works (https://github.com/mimran1980/videoduplicatefinder).
The PR can be merged !

File: VideoDuplicateFinderLinux/DuplicateViewModel.xaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <UserControl xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              
+             x:Class="VideoDuplicateFinderLinux.DuplicateViewModel"
```
Comment (0x90d) [inline VideoDuplicateFinderLinux/DuplicateViewModel.xaml:3]: Nit: format spaces

File: VideoDuplicateFinderLinux/MessageBoxView.xaml.cs [hunk 2]
```diff
@@ -21,7 +21,9 @@ public static class MessageBoxService {
 
 
 	public class MessageBoxView : Window {
-
+		public MessageBoxView() : this("") {
+			
+		}
```
Comment (0x90d) [inline VideoDuplicateFinderLinux/MessageBoxView.xaml.cs:26]: Why was this added?

File: VideoDuplicateFinderLinux/VideoDuplicateFinderLinux.csproj [hunk 3]
```diff
@@ -12,17 +12,24 @@
     <Compile Update="**\*.xaml.cs">
       <DependentUpon>%(Filename)</DependentUpon>
     </Compile>
+<!--
+    <AvaloniaResource Include="**\*.xaml">
+      <SubType>Designer</SubType>
+    </AvaloniaResource>
+    <AvaloniaResource Include="Assets\*"/>
+-->
```
Comment (0x90d) [inline VideoDuplicateFinderLinux/VideoDuplicateFinderLinux.csproj:20]: Why was this added? Please remove

File: VideoDuplicateFinderLinux/VideoDuplicateFinderLinux.csproj [hunk 4]
```diff
@@ -12,17 +12,24 @@
...
     <EmbeddedResource Include="**\*.xaml">
       <SubType>Designer</SubType>
     </EmbeddedResource>
+
```
Comment (0x90d) [inline VideoDuplicateFinderLinux/VideoDuplicateFinderLinux.csproj:24]: Remove

**Review by 0x90d — CHANGES_REQUESTED** [review]
Looks good to me, please see my comments and add the changes so I can merge, thanks!

**Comment by scruffynerf** [comment]
any chance of actually rolling this in, so Linux works in the binary release again?

BTW, it won't build with Avalonia 0.10, Serilog not longer supported.  You have to use 0.9.12

**Comment by 0x90d** [comment]
Thanks for your contribution and sorry for not having this reviewed. Closing because this refers to outdated VDF Version 2

**Pull request closed without merging**